src/Surfnet/StepupMiddleware/ApiBundle/EventListener/ExceptionListener.php

```diff
@@ -93,7 +93,7 @@ private function createJsonErrorResponse(Exception $exception, $statusCode, $hea
         ) {
             $errors = $exception->getErrors();
         } else {
-            $errors = [$exception->getMessage()];
+            $errors = [sprintf('%s: %s', get_class($exception), $exception->getMessage())];
```

Comment:
Pagerfanta, for example, throws exceptions with empty messages. Adding the type helps.